.github/actions/ortdocker/action.yml

```diff
@@ -98,7 +98,7 @@ runs:
       uses: docker/build-push-action@v5
       with:
         context: .
-        platforms: linux/amd64,linux/arm64
+        platforms: linux/amd64
```

Comment:
This commit should only contain the platform changes, all the formatting changes [should be in separate commits](https://github.com/oss-review-toolkit/.github/blob/main/CONTRIBUTING.md):

> [Make separate commits for logically separate changes.](https://github.com/git/git/blob/e6932248fcb41fb94a0be484050881e03c7eb298/Documentation/SubmittingPatches#L43) For example, pure formatting changes that do not affect software behavior usually do not belong in the same commit as changes to program logic.

Reply:
It was a drat before. I updated and added as a PR now.